src/activate/activate.c

```diff
@@ -336,6 +343,7 @@ static int help(void) {
                , program_invocation_short_name
                , link
         );
+        /* clang-format on */
```

Comment:
Please don't add work-arounds like this, and simply exclude this change from the commit.

Reply:
see above, 
I also need a final opinion on these help texts